README.md

```diff
@@ -92,3 +92,22 @@ to find the references to the symbol under point.
 
 ### Flycheck
 ![flycheck](./examples/flycheck.gif)
+
+## Finer Control of Starting lsp-mode
+
+In order to more finely control the `lsp-mode` startup, there are a number of
+customizable variables.
+
+`lsp-project-whitelist` : Defaults to `nil`. If set, `lsp-mode` will only be
+started if the given project root appears in the whitelist.
+
+`lsp-project-blacklist` : Defaults to `nil`. If set, all projects will be
+started except those in this list.
+
+If a project root appears in both lists, the `lsp-project-whitelist` overrides
```

Comment:
That is not right, as I have written above. If the whitelist is used, the blacklist is ignored.